### PR TITLE
Creating architecture diagram of dev environment.

### LINKED ISSUE #69: Creating architecture diagram of dev environment.
**User stories**

As a DevOps Engineer, I want to create an xc3 dev architecture diagram which shows how the Grafana Dashboard will be accessed

**Acceptance Criteria**

- [x] Identify unwanted AWS resources mentioned in prod architecture and remove it in a separate diagram
- [x] Create a new architecture diagram for dev environment
- [x] Add it to the README.md file located inside root directory
- [ ] Review from peers

### PR DESCRIPTION
**Pull request by ghost**
Title: Dev Environment Architecture Diagram
Link: https://github.com/XgridInc/xc3/issues/69

    [PS-2]
- Added details of both 'dev' and 'prod' architecture diagram.

<br>

     [PS-1]
-  Added architecture diagram of dev environment.

Signed-off-by: Kashan Ali [<kashan.ali@xgrid.co>]

### CODE REVIEW

**Review by yaseenzafar123 — APPROVED** [review]
LGTM

**Review by waji46 — APPROVED** [review]
LGTM

**Review by samanXgrid — APPROVED** [review]
LGTM

**Review by yaseenzafar123 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -27,9 +27,13 @@ Check the below video for a quick demo of XC3.
 
 - Provides cost optimization recommendation workflow without exposing your private information
```
Comment (faheem-khau9) [inline README.md:28]: Before this, let's talk about there are two different modes of deployment in XC3

**Review by faheem-khau9 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kashan-01) [reply README.md:28]: Thanks for the feedback. You're right there two modes of deployment in XC3, So ?

**Review by waji46 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kashan-01) [reply README.md:28]: @faheem-khau9 I've added details about 'dev' and 'prod' environment, please have a look. Thanks

**Review by faheem-khau9 — APPROVED** [review]
LGTM

**Review by muhdsaad — APPROVED** [review]
LGTM

**Pull request merged**